manim/constants.py

```diff
@@ -126,3 +126,11 @@ class MyText(Text):
     "you can either start it prior to running Manim or specify the path to the "
     "executable with the --js_renderer_path flag."
 )
+
+# Video qualities
+QUALITIES = {
+    'fourk_quality': 'k',
+    'high_quality': 'h',
+    'medium_quality': 'm',
+    'low_quality': 'l'
+}
```

Comment:
Did you run black on this? In any case, please leave a blank line at the end of the file.